components/video/VideoPlayerView.bs

```diff
@@ -508,7 +509,7 @@ sub onVideoContentLoaded()
     ' Set subtitleTrack property if subs are natively supported by Roku
     selectedSubtitle = invalid
     for each subtitle in m.top.fullSubtitleData
-        if subtitle.Index = videoContent[0].selectedSubtitle
+        if subtitle.Index = videoContent[0].selectedSubtitle or subtitle.IsForced
```

Comment:
If the user selected subtitle track 2, but subtitle track 1 is forced, would this code display subtitle track 1 since it was found first?